docs/sources/alerting/manage-notifications/webhook-notifier.md

```diff
@@ -41,7 +41,9 @@ Example JSON body:
       "silenceURL": "https://play.grafana.org/alerting/silence/new?alertmanager=grafana&matchers=alertname%3DT2%2Cteam%3Dblue%2Czone%3Dus-1",
       "dashboardURL": "",
       "panelURL": "",
-      "valueString": "[ metric='' labels={} value=14151.331895396988 ]"
+      "values": {
+        "data-avg": 14151.331895396988
+      }
```

Comment:
```suggestion
        "values": {
        "B": 44.23943737541908,
        "C": 1
      }
```
I think this example will be more intuitive. Because not many users rename their queries\expressions